locales/en/experimental/create-kafka-instance.json

```diff
@@ -0,0 +1,111 @@
+  "modal_alerts": {
+    "regions_unavailable_message": "Cloud provider regions are temporarily unavailable. Try again later.",
+    "instance_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "over_quota_message": "Your organization has {{maxStreamingUnits}} streaming units available which are already in use. To deploy a new instance, delete an existing one first or <0>contact us</0> to add more streaming units to your account.",
+    "over_quota_title": "Your organization has no streaming units remaining.",
+    "system_unavailable_message": "$t(common:unexpected_error)",
+    "system_unavailable_title": "$t(common:something_went_wrong)",
+    "trial_available_message": "The trial has a duration of {{time}} hours. If you are interested in using this service in production, go to our <0>Streams for Apache Kafka Overview</0> page to see what options are available during creation and learn how to contact us.",
+    "trial_available_title": "You can create a trial instance to evaluate this service.",
+    "trial_unavailable_message": "All available trial instances are currently in use. Try again later.",
+    "trial_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "trial_used_message": "You can deploy 1 instance at a time. To deploy a new instance, delete your existing one first. If you are interested in using this service in production, go to the <0>Streams for Apache Kafka Overview</0> to see what options are available during creation and learn how to contact us.",
+    "trial_used_title": "You already have an instance running"
+  },
+  "no_regions_available": "No regions available",
+  "partitions_value": "up to {{value}}",
+  "quick_start_guide_message": "Need help getting started? Follow our quick start guide.",
+  "quick_start_guide_message_exp": "Follow our quick start guide.",
+  "quick_start_guide_title": "Need help getting started?",
+  "size_field_aria": "More info for Size field",
+  "size_help_content": "Size of a Kafka instance is based on streaming units. The number of streaming units defines the capacity, resources, and limits of an instance. An instance with a larger size can handle higher loads and process larger amounts of events, has more storage, and can handle more clients and connections.",
+  "standard_kafka_size_description": "Learn how to add streaming units to your account",
+  "standard_kafka_streaming_unit": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_zero": "Your organization has no streaming units remaining",
+  "standard_kafka_streaming_unit_one": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_other": "Your organization has {{count}} streaming units remaining",
+  "select_cloud_provider": "Select cloud provider",
+  "select_region": "Select region",
+  "sizes_missing": "Select a cloud provider and region to find the available sizes",
```

Comment:
I think it's rare that this text will display, assuming we will always select default values. But in some of the examples, this message displays when the form is disabled. For now, let's use this message so that we still communicate that the size input is dependent on cloud provider and region selection, without telling them to make a selection they can't make:
> Size options display when a cloud provider and region are selected